frontend/src/store/actions.ts

```diff
@@ -0,0 +1,16 @@
+import { DiscoveryResources } from '../Utils/api-discovery.types';
+
+export const receivedResources = (resources: DiscoveryResources) => ({
```

Comment:
@karelhala so am I correct in that we are still relying on the built in API discovery reducers for these actions? Thats a bit concerning as we could change the reducer without knowing it would break here...

Reply:
I was thinkning of how to send the date from apiDiscovery as well, unfortunately we have to use the reducers and actions from lib-utils, because watch hooks rely on the specific shape. I think if we expose the actions from lib-utils we might be good to go, since they work in atomicity and if we'd ever change how thins work (different action type) we'd still be fine. Plus the actual logic happens in lib-utils - processing of resources and models happens in the lib-utils reducers.